### PR TITLE
Add canonical url to work page

### PR DESCRIPTION
**Pull request by gestchild**
✨ Feature  

## Value
prevents us leaking google juice

### CODE REVIEW

**Comment by derivadow** [comment]
An alternative would be to use a URL scheme something like: .../work/:id#search-string

File: server/views/pages/work.njk [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% extends 'layout/default.njk' %}
 
-{% block pageMeta %}{% endblock %}
+{% block pageMeta %}
+  <link rel="canonical" href="{{ pageConfig.organization.url }}{{ work.urlWithoutQuery }}" />
```
Comment (jamesgorrie) [inline server/views/pages/work.njk]: Could we just set `next.wellcomecollection.org` here, as there's no reason this should be tied to an abstracted out Org?

File: server/controllers/work.js [hunk 2]
```diff
@@ -29,14 +29,15 @@ export const work = async(ctx, next) => {
   const singleWork = await getWork(id);
   const miroId = singleWork.identifiers[0].value;
   const imgLink = imageUrlFromMiroId(miroId, shouldUseIiif(ctx));
+  const urlWithoutQuery = ctx.originalUrl.split('?')[0];
```
Comment (jamesgorrie) [inline server/controllers/work.js]: I think we can get this here?
https://github.com/koajs/koa/blob/master/docs/api/request.md

**Comment by jamesgorrie** [comment]
> An alternative would be to use a URL scheme something like: .../work/:id#search-string

But that doesn't link to anything in the real world.

**Comment by derivadow** [comment]
> But that doesn't link to anything in the real world.

Yes it would e.g. https://next.wellcomecollection.org/works/bxn5trq6#query=watercolour resolves to https://next.wellcomecollection.org/works/bxn5trq6 and (without wishing to get into the httpRange14 thing) is the URI for the work/thing in the real world. 

And because nothing after the hash is passed to the server/ indexed by Google we only have one URI per work.

**Comment by jamesgorrie** [comment]
Ah, apologies, I was missing what you were saying.

It also means that everything we do re: context of where a user is in search would have to be client side.

File: server/views/pages/work.njk [hunk 3]
```diff
@@ -1,6 +1,8 @@
 {% extends 'layout/default.njk' %}
 
-{% block pageMeta %}{% endblock %}
+{% block pageMeta %}
+  <link rel="canonical" href="{{ work.requestOrigin }}{{ work.requestPath }}" />
```
Comment (jamesgorrie) [inline server/views/pages/work.njk]: TO be a complete pedant - we could just `canonicalUri` on the root of the `pageConfig` URL, just a thought as it seems page specific.

**Pull request merged**